Sample/Program.cs

```diff
@@ -135,4 +139,13 @@ private static string WriteFile(string filename, string text)
             return path;
         }
     }
+
+    public class RenderService : HtmlLogFormatter.IViewRenderService
+    {
+        public string RenderToString<TModel>(string template, string title, TModel model)
+        {
+            
+            return RazorEngine.Engine.Razor.RunCompile(template, title, model.GetType(), model);

```

Comment:
I'm not sure I understand this solution. It looks like RazorLight has simply been replaced by RazorEngine. Was that the intention? I thought you wanted to be able to create a production library that didn't include a dependency on Razor.

Reply:
The intention was to remove the dependency from the library, so I used a dependency injection pattern (using the IViewRenderService interface that only implements this method). In the sample program (which is not part of the nuget) I just show that RazorEngine could be used to render the log to string, but anything that implements the interface can be used. And if you like to use RazorLight, go ahead. The Interface doesn't care.

Reply:
I used this as a step to move forward. I removed the dependency on RazorLight from SimpleBrowser and put all of the rendering stuff into a separate library by implementing the interface and setting the renderer in SimpleBrowser. This way, any renderer can be implemented - razor, json, email, etc.